server/schema.ts

```diff
@@ -266,3 +266,10 @@ export interface IAdminTemplate extends ICommonTemplate {
 		};
 	};
 }
+
+export type DataLog = {
```

Comment:
Minor thing, but why did you opt to use a type here instead of an interface? An interface would probably be more consistent with the other definitions already in `schema.ts`.